docs/alincome/jobs.md

```diff
@@ -0,0 +1,191 @@
+---
+id: jobs
+title: |
+    Jobs: ALJob and ALJobList
+sidebar_label: |
+  Jobs
+slug: /alincome/jobs
+---
+
+Asking people about their jobs is a common task when getting financial information. The `ALJob` class tries to be complete but simple to use and answer about, doing the
```

Comment:
Simple for the developer and end user or just for the end user? e.g.
* "...but simple for you to use and simple for your users to answer..."?
* "...but simple for your users to complete..."?

Reply:
Simple for both, but this is another bad usage of the term. I'm being outed as adding the very thing I'm trying to remove, lol